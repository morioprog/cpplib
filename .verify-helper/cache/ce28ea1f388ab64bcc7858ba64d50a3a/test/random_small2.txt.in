1000 1
1 968
1 495
1 648
1 279
1 330
1 95
1 80
1 483
2
1 460
1 115
1 353
1 556
1 236
1 912
1 698
2
1 273
1 951
1 279
1 538
1 681
1 412
1 192
1 477
1 644
1 630
1 753
1 392
1 233
2
1 990
1 901
1 111
2
1 54
1 6
1 922
1 752
1 192
1 414
1 557
1 928
1 526
1 715
1 72
1 762
1 448
1 535
1 784
2
1 86
1 127
1 685
1 537
1 322
1 815
1 797
1 922
2
1 736
1 721
1 742
2
1 122
1 643
1 880
1 890
1 903
1 403
1 243
1 568
1 885
1 552
1 532
1 289
2
1 990
1 838
1 238
2
1 424
1 283
1 503
1 27
1 454
2
1 259
1 262
1 356
1 172
1 161
1 10
1 458
1 436
1 157
1 90
1 350
1 496
1 656
1 165
1 709
1 651
1 371
1 681
1 596
1 548
1 735
1 331
1 714
2
1 265
1 681
1 317
2
2
1 733
1 259
1 250
1 335
1 115
1 67
1 857
1 624
1 218
2
2
1 813
1 438
1 629
1 713
1 713
1 508
1 411
1 790
1 243
1 269
2
1 155
1 769
1 677
1 969
1 150
1 189
1 654
1 395
2
2
1 891
1 85
1 18
1 924
1 694
1 746
1 206
1 165
1 20
1 385
1 690
1 155
1 982
1 902
1 302
1 81
1 128
1 808
1 737
2
1 783
1 735
1 242
1 925
1 134
1 213
1 783
1 892
1 583
2
1 906
1 997
1 116
1 546
2
1 40
2
1 172
1 202
1 624
1 166
1 445
1 694
1 705
1 330
1 906
1 459
2
1 555
1 341
1 569
1 665
1 613
1 275
1 583
1 407
1 818
1 383
1 431
1 687
1 819
1 127
1 972
1 436
1 887
1 471
1 17
1 669
1 951
1 224
1 302
1 204
1 681
1 738
1 162
1 791
1 374
1 143
1 805
1 416
1 724
1 544
1 174
1 660
1 916
2
1 893
1 98
1 410
1 440
1 627
1 986
1 965
1 703
1 596
1 145
1 738
1 493
1 481
1 257
1 358
1 549
1 899
2
1 445
1 277
1 243
1 76
1 672
1 948
1 419
1 813
1 613
1 194
1 97
1 289
1 92
1 583
2
1 263
2
1 222
1 741
1 865
2
2
1 841
1 524
1 695
1 175
1 94
1 66
2
1 695
1 187
1 308
1 751
1 678
2
1 246
1 314
1 450
2
1 836
1 19
1 386
1 339
1 946
1 719
1 852
1 375
1 644
1 973
1 234
1 100
1 711
1 766
1 777
2
1 201
1 451
1 451
1 775
1 754
1 881
1 269
2
1 189
1 543
1 962
1 577
2
1 126
1 243
1 909
1 607
1 513
2
2
1 869
1 804
1 474
1 652
1 869
1 16
1 486
1 63
1 536
1 384
1 470
1 668
1 122
1 546
1 684
1 855
1 756
1 783
1 483
2
1 618
1 673
1 397
1 201
1 12
1 934
1 453
1 558
1 190
1 359
1 90
2
1 502
1 947
1 496
1 576
1 80
1 44
2
1 271
1 792
1 47
1 631
1 705
1 449
1 861
1 273
1 945
1 402
1 767
1 588
1 208
1 679
1 500
1 839
1 593
1 917
2
1 671
1 192
1 791
1 614
2
1 81
1 307
1 660
1 971
1 97
1 229
1 471
1 43
1 84
1 405
1 87
1 536
1 489
1 627
1 19
2
1 268
1 688
1 390
1 574
1 69
1 978
1 611
2
1 171
1 261
1 29
1 81
2
1 699
1 493
1 455
1 425
1 526
1 787
1 277
1 613
1 211
1 16
1 910
1 782
1 533
1 813
1 85
1 782
1 49
1 196
1 286
1 959
1 54
1 215
1 35
1 856
1 151
1 864
1 740
1 230
1 838
1 109
1 156
1 508
1 733
1 699
1 978
2
1 618
1 673
1 20
1 502
1 652
1 815
1 4
1 302
1 205
1 832
1 971
1 563
1 964
1 87
1 377
1 840
1 524
1 75
1 482
2
2
1 477
1 733
1 30
1 4
1 359
1 395
1 862
1 143
1 389
1 509
1 630
1 910
1 188
1 179
1 15
1 750
1 704
1 686
2
1 577
1 647
1 272
1 859
1 109
1 823
2
2
1 167
1 426
1 202
1 392
1 60
1 76
1 824
1 263
1 975
1 178
1 376
1 818
2
1 414
1 405
1 167
2
1 539
2
1 654
1 429
1 368
2
1 270
1 354
1 803
1 794
1 993
1 53
1 575
1 616
1 849
1 78
1 266
1 215
1 622
1 492
1 918
1 684
2
1 532
1 135
2
1 962
1 470
1 645
1 77
1 346
1 822
1 500
2
1 11
2
1 841
1 911
1 507
1 550
1 309
2
1 47
1 755
1 943
1 940
1 418
2
1 119
1 888
1 499
2
1 587
1 810
1 676
1 982
1 483
1 187
2
1 112
1 904
1 946
2
1 629
1 445
1 63
1 991
1 461
1 952
1 909
2
2
1 231
1 140
1 681
1 271
1 159
1 379
1 354
1 363
1 597
1 892
1 576
1 626
1 696
1 881
1 216
1 313
2
1 901
2
1 394
1 394
1 585
1 800
2
1 474
1 732
1 250
1 810
1 130
1 373
2
1 223
1 567
1 206
1 531
1 206
1 919
1 456
1 364
2
1 569
1 604
1 942
1 358
1 977
1 332
2
1 130
1 767
1 642
1 430
1 427
1 575
1 110
1 937
1 75
1 436
1 156
1 443
1 676
1 639
1 403
2
1 936
1 382
1 984
1 812
1 892
1 764
1 135
1 711
1 731
1 379
1 104
1 686
1 269
1 241
2
1 511
1 99
1 766
1 701
1 909
2
1 690
1 934
1 257
1 762
1 722
1 704
2
1 121
1 731
1 640
1 655
2
1 774
1 481
1 674
1 665
2
1 319
1 410
1 505
2
1 708
2
1 628
1 72
1 880
1 150
1 842
1 548
1 562
2
1 572
1 85
1 91
1 30
2
2
1 951
1 345
1 708
1 391
1 793
1 194
2
1 604
1 361
2
1 199
2
1 486
1 440
1 427
1 176
1 314
1 128
1 513
2
1 469
1 762
1 547
1 894
1 429
2
1 261
1 379
2
1 379
1 384
1 274
1 722
1 786
1 749
2
1 386
1 497
1 486
1 707
1 495
1 88
1 84
1 355
1 804
1 83
1 804
1 465
1 557
1 696
1 804
1 612
1 586
2
1 390
2
1 456
1 606
1 145
1 878
1 13
1 609
1 373
1 356
1 917
1 694
2
1 563
1 794
1 908
1 902
1 101
1 74
1 462
1 937
1 99
1 976
1 494
1 205
1 619
1 240
1 754
1 664
1 784
1 1
1 117
1 380
1 93
1 699
1 413
1 206
1 150
1 280
1 47
1 971
1 352
1 721
1 358
1 576
1 720
1 400
1 539
2
1 48
1 699
2
1 148
1 791
1 870
1 235
1 817
2
1 820
1 848
1 772
1 590
1 840
1 68
2
1 169
1 218
1 326
1 771
1 836
1 377
1 468
1 534
1 285
1 621
2
1 289
1 923
1 722
1 535
1 224
1 409
1 583
1 464
1 590
1 828
1 966
1 111
1 731
2
1 973
1 329
1 3
1 655
2
1 584
1 788
1 221
1 430
1 750
1 573
1 978
1 143
2
1 513
1 509
1 915
2
1 976
1 357
1 289
1 564
1 124
1 318
1 202
1 732
1 972
1 66
1 610
2
2
1 930
1 835
1 664
1 619
1 316
1 883
1 213
1 414
1 613
1 340
1 518
1 532
1 883
1 116
1 434
1 722
1 555
1 902
1 807
1 931
1 986
1 358
1 862
1 638
1 795
1 196
1 531
1 667
2
1 272
1 289
1 467
2
1 725
1 945
1 775
1 492
2
2
1 628
1 398
1 821
1 316
1 257
1 838
1 114
1 31
1 752
1 480
1 503
1 829
1 126
2
2
1 607
1 241
1 569
1 517
1 959
1 331
1 664
1 153
1 431
1 670
1 682
1 29
2
1 843
1 899
1 255
1 216
1 311
1 191
1 949
1 451
1 210
1 861
1 849
1 265
1 448
1 130
1 742
1 526
2
1 543
1 177
2
1 653
1 632
1 532
1 863
1 453
1 718
1 926
2
1 943
1 439
1 58
1 319
1 634
1 824
1 310
2
2
1 972
1 843
2
1 239
1 319
1 82
1 301
1 169
