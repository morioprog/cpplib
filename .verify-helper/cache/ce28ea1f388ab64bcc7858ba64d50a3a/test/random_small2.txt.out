80
95
115
111
6
54
72
86
122
27
10
90
127
67
115
157
150
155
18
20
81
40
85
17
98
76
92
97
116
66
94
127
19
100
128
126
134
16
12
44
47
63
19
43
29
16
4
20
4
15
30
35
49
54
60
53
69
75
11
76
47
77
78
80
63
81
81
84
85
87
87
90
75
97
99
104
109
109
110
112
72
30
85
91
119
121
122
128
130
130
83
84
13
1
47
48
68
74
88
3
93
99
66
101
111
116
117
124
31
114
29
126
130
135
58
135
140
